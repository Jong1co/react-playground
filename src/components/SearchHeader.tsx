import React from "react";

const SearchHeader = () => {
  return <div>gd</div>;
};

export default SearchHeader;
