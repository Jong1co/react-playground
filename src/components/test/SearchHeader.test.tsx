import SearchHeader from "../SearchHeader";
import userEvent from "@testing-library/user-event";
import { create } from "react-test-renderer";
import { MemoryRouter, Routes, Route } from "react-router-dom";

describe("search header", () => {
  it("renders correctly", () => {
    const component = create(
      <MemoryRouter>
        <Routes>
          <Route path='/' element={<SearchHeader />} />
        </Routes>
      </MemoryRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
  it("black pink 입력 시 화면에 출력", async () => {});
});
