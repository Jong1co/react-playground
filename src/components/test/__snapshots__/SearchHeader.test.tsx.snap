// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search header renders correctly 1`] = `
<header>
  <form
    action=""
    onSubmit={[Function]}
  >
    <input
      className="css-9egffe css-tmyvc7"
      fontWeight={400}
      onChange={[Function]}
      placeholder="플레이스홀더"
      type="text"
      value=""
    />
    <button>
      검색하기
    </button>
  </form>
</header>
`;
